third_party/terraform/tests/resource_cloudfunctions_function_test.go.erb

```diff
@@ -812,3 +833,58 @@ resource "google_cloudfunctions_function" "function" {
 }
 `, projectNumber, networkName, vpcConnectorName, bucketName, zipFilePath, functionName)
 }
+
+func testAccCloudFunctionsFunction_vpcConnectorUpdated(projectNumber, networkName, functionName, bucketName, zipFilePath, vpcConnectorName string) string {
```

Comment:
I didn't notice any difference between this function and the _vpcConnector one. Is there a difference? If not, let's just reuse that one since it already has the vpcConnectorName as a parameter anyway.

Reply:
Sorry, it was an oversight on my part. 
I have done the needful.